stacks/spica/src/bucket/pages/index/index.component.ts

```diff
@@ -115,19 +115,23 @@ export class IndexComponent implements OnInit {
 
         setTimeout(() => {
           const propertyNameFirst =
-            this.properties.length > 1 && this.properties[1].name
-              ? this.properties[1].name
-              : undefined;
+            this.properties.length > 0 ? this.properties[1].name : undefined;
+          const propertyNameSecond =
+            this.properties.length > 1 ? this.properties[2].name : undefined;
```

Comment:
The condition should be `this.properties.length >= 2`

Reply:
or you can explicitly check if the array has a property at the specified index like `this.properties[2] ? this.properties[2].name : undefined;`

Reply:
I don't need equal operator and condition must be greater than 1